docs/sources/datasources/pyroscope/_index.md

```diff
@@ -24,9 +24,13 @@ weight: 1150
 
 Grafana Pyroscope is a horizontally scalable, highly available, multi-tenant, OSS, continuous profiling aggregation system. Add it as a data source, and you are ready to query your profiles in [Explore][explore].
 
-To learn more about profiling and Pyroscope, refer to the [Introduction to Pyroscope](/docs/pyroscope/introduction/).
+Refer to [Introduction to Pyroscope](/docs/pyroscope/introduction/) to understand profiling and Pyroscope.
 
-For information on configuring the Pyroscope data source, refer to [Configure the Grafana Pyroscope data source](./configure-pyroscope-data-source).
+To use profiling data, you should:
+
+- [Configure your application to send profiles](/docs/pyroscope/latest/configure-client/)
+- [Configure the Grafana Pyroscope data source](./configure-pyroscope-data-source).
+- [View and query profiling data in Explore](./query-profile-data)
```

Comment:
These work and will also work in the Grafana Cloud mount.
Small nit about trailing slashes for consistency.

```suggestion
- [Configure the Grafana Pyroscope data source](./configure-pyroscope-data-source/).
- [View and query profiling data in Explore](./query-profile-data/)
```